src/sql/azdata.proposed.d.ts

```diff
@@ -1790,6 +1795,72 @@ declare module 'azdata' {
 		 * The path of the parent node.
 		 */
 		parentNodePath: string;
+		/**
+		 * list of filters that are supported by the node
+		 */
+		filterDefinitions?: NodeInfoFilterDefinition[];
```

Comment:
`filterableProperties`

Reply:
These aren't necessarily filterable properties. Theoretically someone could have a filter that is based off of something else (say maybe parent node values) - so filterDefinitions makes sense to me.

Reply:
Actually, thinking about it I guess you're probably saying that "this isn't a filter itself, but rather the properties/fields that can be filtered on". So that makes sense - although I don't have a strong opinion either way.